router/pkg/authentication/authentication.go

```diff
@@ -53,6 +55,20 @@ func (a *authentication) Claims() Claims {
 	return a.claims
 }
 
+func (a *authentication) SetScopes(scopes []string) {
+	if a == nil {
+		return
+	}
+	if a.claims == nil {
+		a.claims = make(Claims)
+	}
+	existingScopes := a.claims["scope"]
+	if existingScopes != nil && existingScopes.(string) != "" {
+		scopes = append(scopes, existingScopes.(string))
+	}
+	a.claims["scope"] = strings.Join(scopes, " ")
```

Comment:
Could you reference https://datatracker.ietf.org/doc/html/rfc8693#name-request

Reply:
added a comment with it. I couldn't find an HTTP constant for "scope"